mad_gui/plot_tools/labels/base_label.py

```diff
@@ -75,6 +75,9 @@ def mousePressEvent(self, event: QMouseEvent):  # noqa: N802
             super().mousePressEvent(event)
             self.setPos(self.parent.snap_to_sample(self.pos().x()))
 
+            if event.modifiers() == Qt.ControlModifier:
```

Comment:
The way this is implemented at the moment is that the normal mouse event (Move the label) fires in addition to the rename event if `CTRL` is pressed. Is that intended? Naivly I think it should be either or. So not call `self.setPos` when `CTRL` was pressed. But I could be missing something.